src/NServiceBus.NHibernate.Tests/ApprovalFiles/APIApprovals.Approve.approved.txt

```diff
@@ -3,14 +3,6 @@
 [assembly: System.Runtime.InteropServices.ComVisibleAttribute(false)]
 namespace NServiceBus.Features
 {
-    [System.ObsoleteAttribute("NHibernate gateway persistence is deprecated. Use the new NServiceBus.Gateway.Sql" +
```

Comment:
this says "will be removed in version 10", shouldn't these obsolete types/methods remain for one more major version then?

Reply:
Ah, good catch. I'll fix that.

Reply:
Done in https://github.com/Particular/NServiceBus.NHibernate/pull/470/commits/7cfa75274e7d9a3e94b7eb2ef31bdbf70c081406